packages/twenty-front/src/utils/isNullable.ts

```diff
@@ -0,0 +1,4 @@
+import { isNull, isUndefined } from '@sniptt/guards';
+
+export const isNullable = (value: any): value is null | undefined =>
```

Comment:
We already have a isDefined util, could we use that instead ?

Reply:
isNullable => isNotDefined
isNonNullable => isDefined

Reply:
Hey @lucasbordeau,
Can't seem to find this `isDefined` util?

Reply:
Actually I believe `isDefined` was there but got removed, and `isNonNullable` was introduced. The only new util here is `isNullable` and it makes more sense to define it this way than negating `isNonNullable`

Reply:
For reference : 
- isDefined has been removed but shouldn't have been, we'll fix it thanks !
- isNullable feels wrong because we actually have a TypeScript utility type Nullable<T> and we don't want to check if it's Nullable<MyType> but check if the runtime value is `null | undefined`, that's two separate concepts even if close.